### PR TITLE
Adding Exercise End Point

### PR DESCRIPTION
**Pull request by YahyaAiman**

### CODE REVIEW

File: src/services/exercise/handlers.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      return res.status(400).json({
+        message: responseMessages.error.reqBody,
+      });
+    }
+
+    const insertResult = await prisma.exercise.create({
+      data: inputBody.data,
+    });
+
+    const safeInsertResult = exerciseSchemas.base.parse(insertResult);
```
Comment (day280130) [inline src/services/exercise/handlers.ts]: no need to parse if you only need one property

File: src/services/exercise/handlers.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    return res.status(201).json({
+      message: responseMessages.success.post,
+      id: safeInsertResult.id,
```
Comment (day280130) [inline src/services/exercise/handlers.ts]: simply
```typescript
id: insertResult.id
```

**Review by day280130 — CHANGES_REQUESTED** [review]
please fix

File: src/services/exercise/handlers.ts (hunk 1 above)
Comment (YahyaAiman) [reply src/services/exercise/handlers.ts]: noted, I'll remove it

**Review by day280130 — APPROVED** [review]
LGTM

**Pull request merged**